CHANGELOG.md

```diff
@@ -28,6 +29,198 @@
+  [#4757](https://github.com/Kong/kong/pull/4757)
+
+### Changes
+
+##### Dependencies
+
+- The minimum required OpenResty version has been bumped to `1.15.8.1`.
+  Kong's [openresty-patches](https://github.com/kong/openresty-patches)
+  must still be applied (those patches are already bundled with our official
+  distribution packages) for Kong to work properly.
+  The new OpenResty brings many new features that Kong leverages:
+  1. Kong uses the new [ngx\_http\_grpc\_module](https://nginx.org/en/docs/http/ngx_http_grpc_module.html)
+     to implement it's gRPC proxying support.
+  2. Kong have support of the new [keepalive\_requests](https://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive_requests)
+     directive. It's default value is `100`.
+  3. Kong have support of the new [keepalive\_timeout](http://nginx.org/en/docs/http/ngx_http_upstream_module.html#keepalive_timeout)
+     directive. It's default value is `60s`.
+  4. OpenResty `1.15.8.1` added ARM64 architecture support. With it, Kong now
+     officially supports running on ARM64 as well. ARM64 artifact will be
+     part of official Kong release from now on.
+  5. OpenResty `1.15.8.1` brings better performance when handling incoming
+     requests.
+  6. The LuaJIT GC64 mode is enabled by default in OpenResty `1.15.8.1`,
+     which raised LuaJIT GC-mamaged VM limit from *2 GB* to *128 TB*.
+     In addition, GC64 mode should also result in more predictable GC
+     performance for LuaJIT.
+  [#4382](https://github.com/Kong/kong/pull/4382)
+- Bumped [kong-lapis](https://github.com/Kong/lapis) to `1.7.0.1`.
+  [#4684](https://github.com/Kong/kong/pull/4684)
+- Bumped [luaossl](https://github.com/wahern/luaossl) to `20190612`.
+  [#4735](https://github.com/Kong/kong/pull/4735)
+
+##### Plugins
+- Bumped [kong-plugin-azure-functions](https://github.com/Kong/kong-plugin-azure-functions) to `0.4`.
+  [#4699](https://github.com/Kong/kong/pull/4699)
+- Bumped [kong-plugin-kubernetes-sidecar-injector](https://github.com/Kong/kong-plugin-kubernetes-sidecar-injector) to `0.2`.
+  [#4699](https://github.com/Kong/kong/pull/4699)
+- Bumped [kong-plugin-serverless-functions](https://github.com/Kong/kong-plugin-serverless-functions) to `0.3`.
+  [#4699](https://github.com/Kong/kong/pull/4699)
+- Bumped [kong-prometheus-plugin](https://github.com/Kong/kong-prometheus-plugin) to `0.4`.
+  [#4699](https://github.com/Kong/kong/pull/4699)
+
+### Fixes
+
+##### CLI
+- Fixed an issue if admin ran `kong restart` and Kong was not running,
+  logging would be turned off.
+  [#4772](https://github.com/Kong/kong/pull/4772)
+- Fixed an issue that warning regarding Kong using deprecated config value
+  warnings being shown even when admin is stopping Kong. This ensures such
+  warnings are only shown if admin tries to `start` or `restart` Kong.
+  [#4772](https://github.com/Kong/kong/pull/4772)
+
+##### Core
+- :warning: Fixed a bug in router that when wildcard host name and plain host name
```

Comment:
Use of **that** instead of **where**. Bug description should be in past tense, and fix description should be in present tense (*matches* instead of *will match*). Also, the first description phrase is a bit confusing. Suggested change:

> * Fixed a bug in router **where** if two routes matched the current request host name and one of the routes was a wildcard, the router would fail to match the request. For example: If route 1 was `*.example.com/path1` and route 2 was `plain.example.com/path2`, then a request of `plain.example.com/path1` previously failed to match any of the routes. 
After the fix it correctly matches to route 1.